src/main/core/master.c

```diff
@@ -175,7 +175,13 @@ static gboolean _master_loadConfiguration(Master* master) {
             return FALSE;
         }
 
-        file = utility_getFileContents(fileName->str);
+        // Read config from file or stdin
+        if (0 == g_strcmp0("-", fileName->str)) {
+            file = utility_getFileContent(stdin);
```

Comment:
The names of this new helper function should give more of a clue of how it's different from the other. Maybe `utility_getFileStreamContents`